runelite-client/src/main/java/net/runelite/client/plugins/combatlevel/CombatLevelPlugin.java

```diff
@@ -153,6 +158,35 @@ public void onConfigChanged(ConfigChanged event)
 		}
 	}
 
+	@Subscribe
+	public void onScriptPreFired(ScriptPreFired scriptPreFired)
+	{
+		if (scriptPreFired.getScriptId() == ScriptID.CHARACTER_SUMMARY_TEXT_FORMAT)
+		{
+			final String[] stringStack = client.getStringStack();
+			final int stringStackSize = client.getStringStackSize();
+			final String levelText = Text.removeTags(stringStack[stringStackSize - 1]); // first argument
+
+			// This script is used for both total level and combat level, so verify the script is modifying the combat level
```

Comment:
Are you able to update the widget text after script 3893 (clientscript,account_summary_update_combatlevel) runs rather than needing this check?

Reply:
Unless I'm misunderstanding something about your suggestion, I don't think that works. If you update the widget text in the scriptPostFired event for the script you mentioned, the text winds up off-centered:
<img width="95" alt="off-centered" src="https://user-images.githubusercontent.com/96100526/213347131-7e5970c6-93bd-4389-95d1-7a1017fb27ab.png">

On the other hand, by running with the scriptPreFired event for the 3948 script, the text automatically gets centered as seen in this PR's main description.